met-web/src/components/engagement/form/EngagementWidgets/Map/PreviewModal.tsx

```diff
@@ -29,8 +29,8 @@ export const PreviewModal = () => {
                 >
                     <Map
                         geojson={previewMap.geojson}
-                        longitude={previewMap.longitude}
-                        latitude={previewMap.latitude}
+                        longitude={previewMap.longitude ? previewMap.longitude : undefined}
```

Comment:
Do we need to explicitly pass undefined? when previewMap.longitude fails this ternary it would be already undefined no?